pyproject.toml

```diff
@@ -53,7 +52,7 @@ dependencies = [
   "platformdirs",
   "pydantic >=2.0",
   "pydap >=3.4.0",
-  "pymbolic <=2022.2", # 2024.1.0 is not supported
+  "pymbolic",
```

Comment:
It would be good to have the lowest version that the new vocabulary supports.

Reply:
Agreed, but unclear from a quick look at the release notes. Didn't really have time to hunt the info down. What do you suggest?

Reply:
I need to fix up the formatting, so I'll see what I can do this morning.